server/testutils/containere2e/containere2e.go

```diff
@@ -117,6 +117,7 @@ func NewE2ETestPlugin(t *testing.T, extraOptions ...mmcontainer.MattermostCustom
 	options := []mmcontainer.MattermostCustomizeRequestOption{
 		mmcontainer.WithPlugin(filename, "com.mattermost.msteams-sync", pluginConfig),
 		mmcontainer.WithLogConsumers(&tLogConsumer{t: t}),
+		mmcontainer.WithEnv("MM_EXPERIMENTALSETTINGS_ENABLESHAREDCHANNELS", "true"),
```

Comment:
@wiggin77, can you remind me where things stand in terms of rolling this out to production? Will we require customers to enable this experimental setting (and the enduser functionality) before we can rely on shared channels, or is there a ticket to track decoupling these?

Reply:
Let me remove that, I think I added there while I was trying to figure out the how to fix the test while I was working on the PR, but if I can remove that line and keep going, works for me.